macos/Runner/Base.lproj/MainMenu.xib

```diff
@@ -13,7 +13,7 @@
         </customObject>
         <customObject id="-1" userLabel="First Responder" customClass="FirstResponder"/>
         <customObject id="-3" userLabel="Application" customClass="NSObject"/>
-        <customObject id="Voe-Tx-rLC" customClass="AppDelegate" customModule="Flutter_Gallery" customModuleProvider="target">
+        <customObject id="Voe-Tx-rLC" customClass="AppDelegate" customModule="Runner" customModuleProvider="target">
```

Comment:
Did you mean to change this file? I don't know why the customModule changed from `Flutter_Gallery` to `Runner` or why the `fullSizeContentView` window style mask was removed.

Did you regenerate it, or did Xcode decide to touch the file when you opened it?

Reply:
This was a regeneration. I think we modified both at some point in the past so this is fine